### PR TITLE
playground/pdms: wait for tso component ready

### LINKED ISSUE #2396: tiup playground fail to restart pdms cluster
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do?

After exit from the pdms deployed by tiup playground, rerun playground to restart the cluster failed at tidb starting for no tso service ready.

2. What did you expect to see?

As described in https://asktug.com/t/topic/1023702
tidb failed to start.

image

**What did you expect to see?**

3. What did you see instead?
tiup playground should start tso service before tidb.

5. What version of TiUP are you using (`tiup --version`)?

### PR DESCRIPTION
**Pull request by HuSharp**
### What problem does this PR solve? 
Close #2396

### What is changed and how it works?
can reload cluster with tag
image

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No cod

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

Release notes:

```release-note
NONE
```

### CODE REVIEW

File: pkg/cluster/api/pdapi.go [hunk 1]
```diff
@@ -139,6 +139,7 @@ var (
 	pdStoresURI          = "pd/api/v1/stores"
 	pdStoresLimitURI     = "pd/api/v1/stores/limit"
 	pdRegionsCheckURI    = "pd/api/v1/regions/check"
+	pdMSHealthPrefix     = "api/v1/health"
```
Comment (rleungx) [inline pkg/cluster/api/pdapi.go:142]:
```suggestion
	pdMSHealthPrefix     = "tso/api/v1/health"
```

**Review by rleungx — APPROVED** [review]
The rest LGTM

**Comment by kaaaaaaang** [comment]
please fix the UT

**Comment by HuSharp** [comment]
> please fix the UT

Fixed. But not sure why get `[ERROR] Please install docker-compose` in `integrate-cluster-cmd` :(

**Pull request merged**